GUI/QtQuick/TxInputsModel.cpp

```diff
@@ -166,11 +169,9 @@ int TxInputsModel::setFixedUTXOs(const std::vector<UTXO>& utxos)
    int nbTX = 0;
    for (const auto& utxo : utxos) {
       for (const auto& entry : fixedEntries_) {
-         if ((entry.txId == utxo.getTxHash()) && (entry.txOutIndex == utxo.getTxOutIndex())) {
```

Comment:
This intersection with `fixedEntries_` should be kept, UTXOs can be from different wallet/transaction.